packages/permission-controller/CHANGELOG.md

```diff
@@ -278,7 +278,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
-- Move `SubjectMetadataController` to permission-controller package ([#1234](https://github.com/MetaMask/core/pull/1234))
+- Move `SubjectMetadataController` to permission-controller package ([#5437](https://github.com/MetaMask/core/pull/5437))
```

Comment:
Was this change intentional?

Reply:
No, I was using 1234 as a placeholder until the PR was created 😄

Reply:
Removed.